### ISSUE TITLE
fix accessibility error in nhs_selectizeInput

### ISSUE THREAD

**Issue by kygoffe** [nhsbsa-data-analytics/nhsbsaShinyR#24]
Fix WAVE accessibility missing label error.

**Comment by kygoffe** [comment]
closed with #25
